ga4gh/backends.py

```diff
@@ -217,60 +217,60 @@ def searchVariants(self, request):
             # Normally, we would use a for loop to iterate over the rows in
             # a cursor, but here we use the next() function so that we can
             # end the iteration early when we have generated enough results.
-            r = next(cursor, None)
-            while r is not None and len(v) < request.pageSize:
-                v.append(self.convertVariant(r, sampleRowPositions))
-                r = next(cursor, None)
-            if r is not None:
-                response.nextPageToken = r[self.POS_COL]
+            currentRow = next(cursor, None)
+            while currentRow is not None and len(variants) < request.pageSize:
+                variants.append(self.convertVariant(currentRow, sampleRowPositions))
+                currentRow = next(cursor, None)
+            if currentRow is not None:
+                response.nextPageToken = currentRow[self.POS_COL]
         else:
             # TODO more encoding issues.
             name = request.variantName.encode()
             start = (chrom, name)
             cursor = self._chromIdIndex.cursor(readCols, start)
-            r = next(cursor, None)
+            currentRow = next(cursor, None)
             # for now, we assume that there are 0 or 1 results.
             # TODO are there issues with having several names for a variant?
-            if r is not None:
+            if currentRow is not None:
                 # The result must still be within the range and must match
                 # the specified name exactly. The cursor is positioned at
                 # the first row >= the specified key.
-                if request.start <= r[self.POS_COL] < request.end \
-                        and r[self.ID_COL] == name:
-                    v.append(self.convertVariant(r, sampleRowPositions))
-        response.variants = v
+                if request.start <= currentRow[self.POS_COL] < request.end \
+                        and currentRow[self.ID_COL] == name:
+                    variants.append(self.convertVariant(currentRow, sampleRowPositions))
+        response.variants = variants
         return response
 
     def getMetadata(self):
         """
         Returns a list of GAVariantSetMetadata objects for this variant set.
         """
-        def f(infoField, col):
-            md = protocol.GAVariantSetMetadata()
-            md.key = infoField
-            md.value = ""
-            md.id = ""
+        def getMetadata(infoField, col):
```

Comment:
OK, lets leave it as `f` for now, but try and come up with a better convention for this sort of thing. I don't feel strongly about using/not using single-letter variables, it'll just take me a little getting used to on my part.